src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs

```diff
@@ -2007,51 +2019,54 @@ public void RecursivePattern_01()
                     N(SyntaxKind.CasePatternSwitchLabel);
                     {
                         N(SyntaxKind.CaseKeyword);
-                        N(SyntaxKind.DeconstructionPattern);
+                        N(SyntaxKind.RecursivePattern);
```

Comment:
I didn't see any test in this file that used to produce a `PropertyPatternSyntax` (and should now produce `RecursivePatternSyntax` instead. Are we missing a test case?